Apps/WebClient/src/ClientApp/src/components/navmenu/sidebar.vue

```diff
@@ -234,6 +234,12 @@ export default class SidebarComponent extends Vue {
     }
 
     private get isTimeline(): boolean {
+        if (
+            !this.isMobileWidth &&
+            this.user.preferences.tutorialPopover?.value === "true"
+        ) {
+            this.showTutorialPopover = true;
```

Comment:
this looks odd. Why is the popover variable being set on a query to check what route it is?

Reply:
clicking on Timeline menu should re-evaluate the popover so that we can display it. Do you have an idea to try without doing so?

Reply:
i don't like that his check is here too, i'm thinking there's a another way it can watch for route changes?!

Reply:
mbe I can try to send an event bus from timeline.vue to sidebar.vue to capture this event.

Reply:
I have tried both beforeRouteUpdate and mounted on timeline.vue, they both work only when moving from profile (or another vue) to timeline.vue, but it failed when user login. I will have to investigate further into this.